charts/renga/values.yaml

```diff
@@ -1,49 +1,269 @@
-# Default values for renga.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-
-keycloak:
-  username: admin
-  password: admin
-
-postgresql:
-  postgresPassword: postgres
-  persistence:
-    size: 8Gi
-
-gitlab:
-  password: gitlabadmin
-  persistence:
-    size: 30Gi
+## Default values for renga.
+## This is a YAML-formatted file.
+## Declare variables to be passed into your templates.
 
+## Global variables
+## Shared values/secrets
 global:
-  useHTTPS: false
-  keycloak:
-    postgresDatabase: keycloak
-    postgresUser: keycloak
-    postgresPassword: keycloak
   gitlab:
+    ## Client secret, set this to a random string.
+    ## To change this value after deploying, first generate a new secret using the Keycloak admin console
+    ## then paste the generated secret here
+    clientSecret: open_secret
+    ## Name of the postgres database to be used by Gitlab
     postgresDatabase: gitlabhq_production
+    ## Postgres user for the gitlab database
     postgresUser: gitlab
+    ## Postgres password for the gitlab database
     postgresPassword: gitlab
-    clientSecret: open_secret
-
+  keycloak:
+    ## Name of the postgres database to be used by Keycloak
+    postgresDatabase: keycloak
+    ## Postgres user for the Keycloak database
+    postgresUser: keycloak
+    ## Postgres password for the Keycloak database
+    postgresPassword: keycloak
   renga:
+    ## Domain name for the deployed instance of renga
     domain: example.local
+  ## Set to true if using https
+  useHTTPS: false
 
-externalName:
-  enabled: false
-  target: example.local
-
+## Ingress configuration
+## See: https://kubernetes.io/docs/concepts/services-networking/ingress/
 ingress:
+  ## Enables the creation of an ingress
   enabled: false
+
+  ## Annotations for the created ingress
   annotations:
+    ## The ingress class
     kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
     nginx.ingress.kubernetes.io/proxy-body-size: '0' # Adjust to a reasonable value for production to avoid DOS attacks.
+
+  ## Hosts for the ingress
+  ## Should include at least the value from `global.renga.domain`
   hosts:
   - example.local
+
+  ## TLS setting for the ingress
   tls: []
   #  - secretName: chart-example-tls
   #    hosts:
-  #      - chart-example.local
+  #      - example.local
+
+## Keycloak configuration
+keycloak:
+  ## Keycloak admin user configuration
+  username: admin
+  password: admin
+
+  ## Keycloak image
+  # image:
+  #   pullPolicy: IfNotPresent
+  #   repository: jboss/keycloak
```

Comment:
why is this commented out? presumably because it's set in the keycloak config -- it might be confusing to have it defined in two places?